### PR TITLE
Transform school address from 3 lines to 2 lines for SF submission.

### PR DESCRIPTION
**Pull request by jholton**
## Description of change
GIBFT will not send school `facility_code`.  Transform school address from 3 lines to 2 lines for SF submission.

## Testing done
specs

## Testing planned
manual testing locally/dev/staging

#### Unique to this PR
- [x] remove school lookup via `facility_code`
- [x] condense school address to 2 lines if 3 lines are popluated

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by lihanli** [comment]
@rianfowler has the issue with feedback tool not being able to save school address in save in progress been resolved?

**Comment by rianfowler** [comment]
@lihanli I just tested SiP for the school address locally and it restored both the searched school and the manually entered school. We started saving the facility code as a `view:` attribute along with some other stuff that allows to restore the search.

File: app/models/gi_bill_feedback.rb [hunk 1]
```diff
@@ -75,17 +59,14 @@ def transform_form
     transformed.merge!(get_user_details)
 
     transformed['education_details'].tap do |education_details|
-      next if education_details.blank?
```
Comment (lihanli) [inline app/models/gi_bill_feedback.rb]: why is this deleted?

Comment (jholton) [reply app/models/gi_bill_feedback.rb]: since `location_code` is no longer an option, `education_details` is required/popluated on the frontend whether via school search or manual entry.
[Required in the schema too.](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/FEEDBACK-TOOL/schema.js#L134)
